faststream/confluent/prometheus/middleware.py

```diff
@@ -0,0 +1,25 @@
+from typing import TYPE_CHECKING, List, Optional
+
+from faststream.confluent.prometheus.provider import settings_provider_factory
+from faststream.prometheus.middleware import BasePrometheusMiddleware
+
+if TYPE_CHECKING:
+    from prometheus_client import CollectorRegistry
+
+
+class KafkaPrometheusMiddleware(BasePrometheusMiddleware):
+    def __init__(
+        self,
+        *,
+        registry: "CollectorRegistry",
+        app_name: str = "faststream",
+        metrics_prefix: str = "faststream",
```

Comment:
`metrics_prefix` should use `app_name` as default if unset

Reply:
fixed

Reply:
You forgot about real broker classes. They don't use `EMPTY` as default

Reply:
fixed